src/components/molecules/Dropdown/Dropdown.js

```diff
@@ -51,6 +51,9 @@ export default class Dropdown extends HTMLElement {
       this.querySelector(
         'cod-button[data-bs-toggle="dropdown"]',
       ).addEventListener('click', this._onClick);
+      this.querySelector(
+        'cod-button[data-bs-toggle="dropdown"]')
+      .setAttribute('aria-controls', '#dropdown-id');
```

Comment:
We'll have to follow a similar approach described below on line 84 to get the dropdown ID.